website/blog/2022-01-31-hello-indigo-game.md

```diff
@@ -0,0 +1,154 @@
+# Converting `IndigoSandbox` to `IndigoGame`
```

Comment:
You'll need to add the headers, e.g.:
```
---
title: Post-Mortem: "RoguelikeDev Does The Complete Roguelike Tutorial" in Scala
author: Dave Smith
authorURL: http://twitter.com/davidjamessmith
---
```
I'm sure docusaurus will have a "draft" flag too.